packages/nx-plugin/src/generators/library/index.ts

```diff
@@ -1,12 +1,7 @@
 import { formatFiles, Tree, writeJson } from '@nrwl/devkit';
```

Comment:
You can remove the related test cases here: https://github.com/swarmion/swarmion/blob/4c52427f180083796344bc1ac4f8ce582c4e968f/packages/nx-plugin/src/generators/library/library.test.ts#L51

Reply:
How come the test did not fail ? 👀